ops/terraform/modules/resources/bfd_pipeline/main.tf

```diff
@@ -0,0 +1,60 @@
+# EC2 Instance to run the BFD Pipeline app.
+#
+module "ec2_instance" {
+  source = "../ec2"
+
+  env_config      = var.env_config
+  role            = "etl"
+  layer           = "data"
+  az              = "us-east-1b" # Same as the master db
+
+  launch_config   = {
+    instance_type = "m5.2xlarge"
+    volume_size   = 100 # GB
+    ami_id        = var.launch_config.ami_id
+
+    key_name      = var.launch_config.ssh_key_name
+    profile       = var.launch_config.profile
+    user_data_tpl = "pipeline_server.tpl"
+    git_branch    = var.launch_config.git_branch
+    git_commit    = var.launch_config.git_commit
+  }
+
+  mgmt_config     = {
+    vpn_sg        = var.mgmt_config.vpn_sg
+    tool_sg       = var.mgmt_config.tool_sg
+    remote_sg     = var.mgmt_config.remote_sg
+    ci_cidrs      = var.mgmt_config.ci_cidrs
+  }
+}
+
+# Security group for application-specific (i.e. non-management) traffic.
+#
+resource "aws_security_group" "app" {
```

Comment:
I assume this SG is made for the ETL server instance. Right? How does it get attached to the instance?

Reply:
The power of positive thinking?

Reply:
(Very good catch! Updated the config and plan.)